bin/setup

```diff
@@ -17,6 +17,20 @@ Dir.chdir APP_ROOT do
   #   system "cp config/database.yml.sample config/database.yml"
   # end
 
+  puts "\n== Creating Template Database =="
+  if system "command -v createdbn"
```

Comment:
You mean `createdb`, not `createdbn` here, right?

Reply:
Oh, fingers. Fixed. :-)